### ISSUE TITLE
Models Zoo is empty 

### ISSUE THREAD

**Issue by LebToki** [ParisNeo/lollms-webui#273]
## Expected Behavior
it failed to produce a list of models

## Current Behavior
moved models manually to the models folder but still failed to show

## Context
Windows 11

**Comment by ParisNeo** [comment]
When you install this the first Time you got prompted to where to put your data. If you chose the default option it is installed on your documents folder/lollms
Open it go to configs then local_config. You should see that model_name is null.
Replace that with the name of your model and retry. Don't forget the .bin

**Comment by LebToki** [comment]
Milles Merci
I think you should include this in the setup script or at least have a notice somewhere in there.
Next in line I think is choosing a Binding under Windows 11 for some reason.
will update you further on the progress !

**Comment by ParisNeo** [comment]
Yes, I just finished adding this so I didn't have time to add the documentation yet.
